<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CRM - Login</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_left">
    <property name="geometry">
     <rect>
      <x>-50</x>
      <y>0</y>
      <width>301</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:  rgb(40, 60, 130);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="logo_lbl">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>151</width>
       <height>161</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>logo-icon.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="logolbl_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>250</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../images/logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>420</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #fff</string>
     </property>
     <property name="text">
      <string>V6-G2</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>430</width>
      <height>530</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 180, 200);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="username_lbl">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ffffff</string>
     </property>
     <property name="text">
      <string>User Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username_inp">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 6px; 
background-color:rgb(239, 239, 239);
border-radius: 10px</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_inp">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>240</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">padding:6px;
background-color:rgb(239, 239, 239);
border-radius:10px</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="exit_btn">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>360</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgb(40, 60, 130);
color: #fff
}
QPushButton:hover {
background-color: rgb(218,30,60)
}</string>
     </property>
     <property name="text">
      <string>   Exit</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/exit.svg</normaloff>../images/exit.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="login_btn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>360</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: rgb(40, 60, 130);
color: #fff
}
QPushButton:hover {
background-color: rgb(218,30,60)
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="mesaj_lbl">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>310</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>9</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Crmlogin_lbl">
     <property name="geometry">
      <rect>
       <x>74</x>
       <y>40</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#fff</string>
     </property>
     <property name="text">
      <string>CRM LOGIN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>20</width>
       <height>191</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="password_lbl">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Reference Sans Serif</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #ffffff</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tr_TR">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/flag-for-flag-turkey.svg</normaloff>../images/flag-for-flag-turkey.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="nl_NL">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>110</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/netherlands-holland.svg</normaloff>../images/netherlands-holland.svg</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="en_EN">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>110</y>
       <width>41</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/united-kingdom-uk.svg</normaloff>../images/united-kingdom-uk.svg</iconset>
     </property>
    </widget>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame_left</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
